src/Client.php

```diff
@@ -172,6 +172,15 @@ public function getEndpoint(): string
     public function getUserAgent(): string
     {
         if (null !== $this->userAgentPrefix) {
+            $userAgent = explode('~', $this->userAgentPrefix);
+            if (count($userAgent) > 1) {
+                if ('' === $userAgent[1]) {
+                    return sprintf("{$userAgent[0]} ({$this->getClientVersion()}; {$userAgent[2]}; {$this->getPHPVersion()})");
+                }
+
+                return sprintf("{$userAgent[1]} ({$userAgent[0]}; {$this->getClientVersion()}; {$userAgent[2]}; {$this->getPHPVersion()})");
```

Comment:
Incorrect usage of `sprintf()`, `{$foo}` works without `sprintf()`, Use variable replacements, then `$this->getPHPVersion()` method also becomes unnecesary.)